### DISCUSSION TITLE
Break point in margin not working.

### DISCUSSION THREAD

**Discussion by RazneshPrajapati** [quasarframework/quasar#16549] [General - Components / Directives / etc] [answered]
I am using quasar@2.13.0 and while using break point in margin its not working.

```
<div class="row justify-center q-mb-lg q-mb-lg-xl q-col-gutter-lg">
</div>
```

I have done like this. Isn't this the correct way? I have followed according to document.
Can anyone help me on this?

**Comment by metalsadman** [comment, marked as answer]
have you set `framework>cssAddon: true`? https://quasar.dev/style/spacing#flex-addons, and you have several margins there that are maybe canceling out/overriding each other.

**Reply by RazneshPrajapati** [reply]
Thank you for your help. I missed that.
